package.json

```diff
@@ -9,14 +9,16 @@
     "test:demo:browser": "yarn --cwd examples/express && yarn --cwd examples/express test",
     "test:demo:nodejs": "node examples/node/index.js",
     "test:demo:esm": "yarn --cwd examples/esm && yarn --cwd examples/esm start",
-    "build:playground:vue": "yarn --cwd ./playgrounds/vue && yarn --cwd ./playgrounds/vue build",
-    "build:playground:react": "yarn --cwd ./playgrounds/react && yarn --cwd ./playgrounds/react build",
+    "build:playground:vue": "yarn build && yarn --cwd ./playgrounds/vue && yarn --cwd ./playgrounds/vue build",
+    "build:playground:react": "yarn build && yarn --cwd ./playgrounds/react && yarn --cwd ./playgrounds/react build",
+    "build:playground:vanilla-js": "yarn build && yarn --cwd ./playgrounds/vanilla-js && yarn --cwd ./playgrounds/vanilla-js build",
     "test:all": "yarn test && yarn test:demo",
     "lint": "eslint --ext .js,.ts,.tsx,.vue .",
     "lint:fix": "eslint --ext .js,.ts,.tsx,.vue --fix .",
     "build": "rollup -c rollup.config.js && rollup --environment NODE_ENV:production -c rollup.config.js ",
-    "playground:vue": "yarn --cwd ./playgrounds/vue && yarn --cwd ./playgrounds/vue serve",
-    "playground:react": "yarn --cwd ./playgrounds/react && yarn --cwd ./playgrounds/react start"
+    "playground:vue": "yarn build && yarn --cwd ./playgrounds/vue && yarn --cwd ./playgrounds/vue serve",
+    "playground:react": "yarn build && yarn --cwd ./playgrounds/react && yarn --cwd ./playgrounds/react start",
```

Comment:
the `build` step is not necessary because the built files are not needed for the react and vue playgrounds